index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400&display=swap" rel="stylesheet">
+    <title>Muneeb's protfolio</title>
+</head>
+<body>
+    <!--This is the header section-->
+    <header>
+        <nav>
+            <div class="Navbar">
+            <a href="#"><img src="req/mylogo.png" alt="logo" id="mylogo"></a>
+            <!--Nav bar button-->
+            <img id="logo" src="req/Union.png" alt="logo">
+            </div>
+        </nav>
+    </header>
+    <!--This is Headline-->
+    <section class="headline">
+    <div>
+        <h1 id="Headline-Title">I’m Muneeb<br>
+            Glad to <br>see you!</h1></div>
+    <div>
+        <p id="Headline-Text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.  </p>
+    </div> 
+    <div class="socialmedia">
+        <div class="helpingtext">LET'S CONNECT</div>
```

Comment:
- [ ] You've done a great job so far on this project, but you still need to improve on something for it to be approved. I noticed that you are using a `div` tag here while said on this project's requirements that The subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`, kindly use `hx` instead of `div` tag for semantically reason